README.md

```diff
@@ -77,3 +78,6 @@ Install it with:
 `sudo gem install gherkin_lint`
 
 After that `gherkin_lint` executable is available.
+
+## Configuration
+If you have a custom configuration you'd like to run on a regular basis instead of passing enable and disable flags through the CLI on every run, you can configure a ```.gherkin_lint.yml``` file that will be loaded on execution.  The format and available linters are in [```config/default.yml```](config/default.yml)
```

Comment:
We took a look at this in rubocop and there is an lot of code that makes this work.  We figured that this would be an acceptable shortcut for now.  You don't have to have .gherkin_lint and the only people who need to know where the default is are gherkin_lint developers.  I agree that there needs to be a better way to add new linters, but we didn't want to take that architectural liberty.

That said, I'll see if there is a quick and dirty way to solve this